src/bot/functions/rotate-barrel-to.js

```diff
@@ -27,22 +29,16 @@ module.exports = function(degreeAngle, callback) {
 
 function rotateBarrelTo(angle, elapsedMs) {
   let r = this.barrel.rotationVelocity * elapsedMs;
-  if ((this.barrel.rotation < angle && this.barrel.rotation + r >= angle) || (this.barrel.rotation > angle && this.barrel.rotation + r <= angle)) {
+  if ((this.barrel.rotation <= angle && this.barrel.rotation + r >= angle) || (this.barrel.rotation >= angle && this.barrel.rotation + r <= angle)) {
     this.barrel.rotationVelocity = 0;
     this.barrel.rotation = angle;
 
-    // standardize angle to between 0 and Math.PI*2
-    while (this.barrel.rotation > Math.PI*2) {
-      this.barrel.rotation -= Math.PI*2;
-    }
-    while (this.barrel.rotation < 0) {
-      this.barrel.rotation += Math.PI*2;
-    }
-
+    this.barrel.rotation = utils.standardizeAngle(this.barrel.rotation);
     return { finished: true };
   }
   else {
     this.barrel.rotation += r;
+    this.barrel.rotation = utils.standardizeAngle(this.barrel.rotation);
```

Comment:
In the case that `angle = pi/32`  and `this.barrel.rotation = 1.9*Pi`

The fastest way to reach the specified angle is to increase `this.barrel.rotation`. Without standardizing, the rotation will never be within +-r.